common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java

```diff
@@ -16,16 +16,21 @@
 
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.browser.Browser;
 import org.eclipse.swt.events.DisposeEvent;
 import org.eclipse.swt.events.DisposeListener;
+import org.eclipse.swt.internal.cocoa.NSBundle;
```

Comment:
This is likely not to compile on non MacOS platforms

Reply:
@jeffmaury why not? All OS specific fragments are present at compile time, they're only filtered out on runtime. Btw. jenkins compiled it. What am I missing?

Reply:
Maybe Tycho is using all fragments but not Eclipse. It tried to load the PR in Eclipse and I got compilation failures

Reply:
@jeffmaury turned to use reflection. Please test.